ctypeslib/codegen/codegenerator.py

```diff
@@ -462,7 +462,22 @@ def Enumeration(self, tp):
         for item in tp.values:
             self._generate(item)
         if tp.name:
-            print("%s = ctypes.c_int # enum" % tp.name, file=self.stream)
+            # Enums can be forced to occupy less space than an int if possible:
+            #  1) Add the attribute `__attribute__((__packed__))` to the C variable declarations
+            #  2) Set the compiler flag ` CFLAGS += -fshort-enums`
+            # In any case, we should trust the enum size returned by the compiler.
+            # https://stackoverflow.com/a/54527229/1641819
+            # https://stackoverflow.com/a/56432050/1641819
+            if tp.size == 1:
```

Comment:
I believe this mapping already exists somewhere else in the library. This is more a proof on concept.